Makefile.am

```diff
@@ -37,7 +36,8 @@ openpower_proc_control_CXXFLAGS = $(PHOSPHOR_LOGGING_CFLAGS) \
 
 openpower_proc_nmi_LDFLAGS = $(PHOSPHOR_LOGGING_LIBS) \
                              $(PHOSPHOR_DBUS_INTERFACES_LIBS) \
-                             $(SDBUSPLUS_LIBS)
+                             $(SDBUSPLUS_LIBS) \
```

Comment:
Update commit message based on the new flow.

Reply:
Updated